main.go

```diff
@@ -98,10 +101,10 @@ func main() {
 	namespaceSelector := cache.ByObject{
 		Field: fields.ParseSelectorOrDie(fmt.Sprintf("metadata.namespace=%s", operatorNamespace)),
 	}
+
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:             scheme,
 		MetricsBindAddress: *metricsAddr,
```

Comment:
@fedepaol I assume this version of controller-runtime doesn't allow you to do the same for metrics ?

Reply:
right, plus the default value for metrics is :0 (meaning, we don't listen to metrics)